src/test/credentials/provider/ssoAccessTokenProvider.test.ts

```diff
@@ -0,0 +1,230 @@
+/*!
+ * Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import * as assert from 'assert'
+import * as AWS from 'aws-sdk'
+import * as sinon from 'sinon'
+import { SSOOIDC } from 'aws-sdk'
+import { DiskCache } from '../../../credentials/sso/diskCache'
+import { SsoAccessTokenProvider } from '../../../credentials/sso/ssoAccessTokenProvider'
+import { CreateTokenResponse, StartDeviceAuthorizationResponse } from 'aws-sdk/clients/ssooidc'
```

Comment:
You have three `aws-sdk` imports. Consolidate to one (maybe two for the `aws-sdk/clients/ssooidc` import)

Reply:
Done